contracts/asserts.cairo

```diff
@@ -0,0 +1,31 @@
+use contracts::dummy_syscalls;
+
+fn assert_only_self() {
+    let self = dummy_syscalls::get_contract_address();
+    let caller_address = dummy_syscalls::get_caller_address();
+    assert(self == caller_address, 'argent: only self');
+}
+
+fn assert_non_reentrant(signer: felt) {
+    let caller_address = dummy_syscalls::get_caller_address();
+    assert(caller_address == 0, 'argent: argent: no reentrant call');
+}
+
+
+fn assert_initialized(signer: felt) {
```

Comment:
Given that it doesn't read storage, this assert doesn't really abstract anything so maybe we should move it to account contract. But let's leave it here for now and decide on that later.